src/schema/v1.json

```diff
@@ -487,6 +489,12 @@
          "title":"DigitalOcean",
          "$ref": "#/definitions/artifact"
         },
+       "ibmcloud": {
+         "$id":"#/properties/images/properties/ibmcloud",
+         "type":"object",
+         "title":"ibmcloud",
```

Comment:
the `titles` appear to usually be capitalized, but not sure if that is required

Reply:
Updated it to `IBM Cloud`